File: 9781430238010/9781430238010_Ch12/FileTest.java
import java.io.*;
import java.util.*;

public class FileTest
{
	public static void main(String [] args)
	{
		File fd = new File("NotAFile.txt");
		System.out.println("File exists? " + fd.exists());

		try {
			FileReader fr = new FileReader(fd);
		} catch (FileNotFoundException e) {
			System.out.println("NotAFile.txt " + e.getMessage());
		}
	}

}

File: 9781430238010/9781430238010_Ch12/fileTest.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char **argv)
{
	FILE *fd;
	char *fname = "NotAFile.txt";

	if ((fd = fopen(fname, "r")) == NULL) {
		perror("File not opened");
		exit(1);
	}
	printf("File exists\n");
	return 0;
}
